src/popup/components/Protocol/EnsureUserSavedMnemonic.tsx

```diff
@@ -47,28 +55,56 @@ const WordButton = (props: {
   onClick: () => void;
 }) => {
   return (
-    <Button disabled={!props.isEnabled} onClick={props.onClick}>
+    <Button
+      className={(() => {
+        return !props.isEnabled ? "ClickedWord" : "";
+      })()}
+      onClick={props.onClick}
+    >
       {props.word}
     </Button>
   );
 };
 
-export function EnsureUserSavedMnemonic(props: { onComplete: () => void }) {
-  const [buttons, setButtons] = React.useState<CheckButton[]>([]);
-  const [mnemonicArr, setMnemonicArr] = React.useState<string[]>([]);
+const TextField = (props: { input: string }) => {
+  return <TextArea defaultValue={props.input} />;
+};
 
-  const [counter, setCounter] = React.useState(0);
+export function EnsureUserSavedMnemonic(props: { onComplete: () => void }) {
+  const [pageOverride, setPageOverride] = useState<JSX.Element | null>(null);
+  const [buttons, setButtons] = useState<CheckButton[]>([]);
+  const [success, setSuccess] = useState(false);
```

Comment:
I think this can be inferred by the expression you have on line 98/99. You can have that expression in the top level of this function.

```
const [currentInput, setCurrentInput] = useState<string[]>([]);

const success == currentInput.join(" ") === mnemonic;
```